ckanext/ontario_theme/templates/internal/package/read.html

```diff
@@ -1,8 +1,28 @@
-{% ckan_extends %}
+{% extends "package/read_base.html" %}
 
+{% block primary %}
+<article class="module package-z-index">
+  <header class="page-header col-sm-8">
```

Comment:
```suggestion
  <header class="page-header col-xs-12 col-sm-8">
```
To allow aside to be full width

Reply:
Added as part of the solution above, thanks!